docs/recipes.rst

```diff
@@ -182,3 +182,23 @@ as the deploy repo running ``doctr configure``. When deploying, use
 The deploy key for pushing to a wiki is the same as for pushing to the repo
 itself, so if you are pushing to both, you will not need more than one deploy
 key.
+
+Separating source and output in Github username and orgname pages
+=================================================================
+
+Github allows users to create pages at the root url of users or organizations
+http://github.io pages. For example an organization ``coolteam`` can setup a
+repository with the name ``coolteam.github.io`` and the html files in the
+master branch of this repository will be served to https://coolteam.github.io.
+If you are using a static site generator you may want to separate your web
+site's source files from the generated output. To do this, create a new branch,
```

Comment:
Actually as you saw, currently with doctr you have to do this. Either put the sources on a separate branch or on a separate repo (there's already [a recipe](https://drdoctr.github.io/recipes.html#deploy-to-a-separate-repo) for separate repo).

Reply:
You don't have too. I first had the sources and output files mixed all in the master branch. It works but you can possibly get clashes if you aren't careful with file and directory names, etc.

Reply:
Oh yeah, but yes the infinite loop was probably going in the sources + outputs in master branch. So you shouldn't do that.

Reply:
With https://github.com/drdoctr/doctr/pull/318 you will be able to, but even then it's not ideal because every commit will result in a redundant build.

Reply:
Plus if you put your source files on `master` GitHub will serve all your source files on GitHub pages. That probably isn't a big deal but it could be unexpected.

Reply:
Yeah. I changed the wording to point the user away from doing this.